ui/src/routes/conversations/[id]/ConversationMessageInput.svelte

```diff
@@ -89,15 +102,17 @@
       </div>
     </label>
 
-    <div class="flex w-full flex-col">
+    <div class="flex w-full flex-col overflow-y-auto leading-6 max-h-[7.5rem]">
```

Comment:
```suggestion
    <div class="flex w-full flex-col max-h-[7.5rem]">
```
Seems like neither of these rules do anything, unless I'm missing something